### ISSUE TITLE
Buggy and Freezing

### ISSUE THREAD

**Issue by spartandrew18** [AnKing-VIP/anki-audiovisual-feedback#9]
Hello @AnKingMed,

Hope the new year kicks off well for you. I know you are very busy with residency but whenever you have the chance, would you mind taking a look at the performance of this addon.

I am on Anki 23.12.1

I have noticed that when I am doing my cards for the AnKing deck, the addon will either freeze up on the card or just advance without showing the answer. This seems to be replicated most when enabling third party pictures (I.e. sketchy) to be automatically shown on answering the card. Turning off the addon seems to resolve the issue.

Sincerely,
Andrew

**Comment by AnKingMed** [comment]
My suspicion is it's due to the known issues with the video drivers. Try changing to a different driver in the preferences and restarting.  This Will hopefully be fixed in future Anki versions.

**Comment by spartandrew18** [comment]
> My suspicion is it's due to the known issues with the video drivers. Try changing to a different driver in the preferences and restarting. This Will hopefully be fixed in future Anki versions.

I have tried changing to Direct3D and OpenGL and the same occurred. I figured the other are way to slow to be enjoyable.

**Comment by AnKingMed** [comment]
Agreed. Lets just wait and see if things improve in future versions. Unfortunately I don't think there's much we can do about this one
